### DISCUSSION TITLE
why Github Copilot is trying to charge me?

### DISCUSSION THREAD

**Discussion by mrxdaniel21** [community/community#127942] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

why Github Copilot is trying to charge me a payment if i have a yealy plan and i already payed it

**Comment by Alfathull** [comment]
If GitHub Copilot is attempting to charge you despite having already paid for a yearly plan, it could be due to a few potential issues. Here are some steps to troubleshoot and resolve this:

### 1. **Verify Your Payment Status**

- **Check Billing History:**
  Log in to your GitHub account and navigate to your billing settings. Verify that your payment for the yearly plan has been processed and there are no outstanding invoices.

  - Go to [GitHub Billing Settings](https://github.com/settings/billing).

### 2. **Confirm Subscription Details**

- **Subscription Confirmation:**
  Ensure that your subscription status shows as active for the yearly plan. Sometimes, subscription details may not update correctly.

  - Go to [GitHub Copilot Settings](https://github.com/settings/copilot) and check your subscription details.

### 3. **Contact GitHub Support**

- **Submit a Ticket:**
  If everything appears correct but you are still being prompted for payment, it’s best to contact GitHub support directly. Provide them with your payment details, subscription confirmation, and any relevant transaction IDs.

  - Use the [GitHub Support Contact Form](https://support.github.com/contact) to submit a ticket. Make sure to select the appropriate topic area (e.g., "Billing & Payments").

### 4. **Check for Multiple Accounts**

- **Multiple Accounts:**
  Verify that you are logged into the correct GitHub account. If you have multiple GitHub accounts, it’s possible that the payment was applied to a different account than the one you are currently using.

### 5. **Review Communication from GitHub**

- **Emails and Notifications:**
  Check your email for any communication from GitHub regarding billing issues, payment failures, or subscription renewals. Sometimes, notifications may provide additional details on why a charge is being attempted.

### 6. **Possible Technical Glitch**

- **Technical Issues:**
  Occasionally, technical glitches can occur. If you suspect this might be the case, reporting the issue to GitHub support will help them identify and resolve any backend problems.

### Example Message to GitHub Support

Here’s a template you can use when contacting GitHub support:

```plaintext
Subject: Billing Issue with GitHub Copilot Yearly Plan

Dear GitHub Support,

I am writing to report an issue with my GitHub Copilot subscription. Despite having paid for a yearly plan, I am being prompted to make an additional payment.

Details:
- GitHub Username: [Your Username]
- Email Associated with Account: [Your Email]
- Transaction ID: [Payment Transaction ID]
- Date of Payment: [Date of Payment]

Could you please verify my payment status and resolve this issue? Thank you for your assistance.

Best regards,
[Your Name]
```

By following these steps, you should be able to identify and resolve why GitHub Copilot is trying to charge you despite having a yearly plan already paid.

**Comment by ebndev** [comment]
Hi @mrxdaniel21,

Please open a ticket on our [Support page](https://support.github.com/contact) for billing related issues!

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
